packages/chain-adapters/src/bitcoin/utxoSelect.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-console */
+import * as unchained from '@shapeshiftoss/unchained-client'
+import coinSelect from 'coinselect'
+import split from 'coinselect/split'
+
+import { bitcoin } from '../types'
+
+export type MappedUtxos = Omit<unchained.bitcoin.Utxo, 'value'> & { value: number }
+
+const handleStandardTx = ({
```

Comment:
the name `handle` is usually used for functions that handle events. Is there a more descriptive name of what this is actually doing?  I'm not entire sure what it's doing.